Dockerfile

```diff
@@ -21,7 +21,7 @@ COPY pyproject.toml MANIFEST.in README.md LICENSE /src/
 WORKDIR /src
 
 # install requirements
-RUN --mount=type=cache,target=/root/.cache \
+RUN --mount=type=cache,target=/root/.cache,Z \
```

Comment:
Tested this with `Docker version 27.2.0, build 3ab4256` on MacOS and received a:

```
Dockerfile:24
--------------------
  23 |     # install requirements
  24 | >>> RUN --mount=type=cache,target=/root/.cache,Z \
  25 | >>>     python -c 'import toml; print("\n".join(toml.load("./pyproject.toml")["project"]["dependencies"]));' > requirements.txt && \
  26 | >>>     pip install -r requirements.txt
  27 |
--------------------
ERROR: failed to solve: invalid field 'Z' must be a key=value pair
```

I was not able to find the `Z` syntax in the docker documentation, can you please point me to it?

Reply:
Found the documentation for `:Z` https://docs.docker.com/engine/storage/bind-mounts/#configure-the-selinux-label .

For the sake of maximum compatibilty maybe I should just drop the `mount=type=cache` from the Dockerfile? I added this when I was on the road with poor internet.